Ontologies.Mappings/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json

```diff
@@ -166,6 +262,18 @@
       "InputDtmi": "dtmi:mapped:core:Condenser_Water_Supply_Temperature_Setpoint;1",
       "OutputDtmi": "dtmi:com:willowinc:EnteringCondenserWaterTemperatureSetpoint;1"
     },
+    {
+      "InputDtmi": "dtmi:mapped:core:Connected_Access_Point_Status;1",
+      "OutputDtmi": "dtmi:com:willowinc:State;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Connected_Devices_Count_Status;1",
+      "OutputDtmi": "dtmi:com:willowinc:State;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Controller;1",
+      "OutputDtmi": "dtmi:com:willowinc:MotorController;1"
```

Comment:
Equipment;1 if we have no idea what kind of controller this is? Shouldn't guess Motor without any evidence.

Reply:
Agreed that Motor is not a fair assumption. Mapped introduced this concept to represent a Controller, which controls equipment. Granted, this concept is not currently utilized in practice. I will map this to Equipment for now.